client/mksubvolume.cc

```diff
@@ -144,20 +149,41 @@ do_add_fstab_and_mount(MntTable& mnt_table, const libmnt_fs* fs, const string& s
     mnt_fs_set_options(x, options);
     free(options);
 
-    // Caution: mnt_context_mount may change the source of x so the fstab
-    // functions must be called first.
-    mnt_table.add_fs(x);
-    mnt_table.replace_file();
+    return x;
+}
+
+
+void
+do_add_fstab_and_mount(MntTable& mnt_table, libmnt_fs* x)
+{
+    if (verbose)
+	cout << "do-add-fstab-and-mount" << endl;
```

Comment:
I know this line was already there, but what's the purpose of this? Is it a debugging message or something for the end user? For debugging it should use `y2deb`/`y2mil`, for end users it should contain a better text and should be translated...